src/api/app/models/review.rb

```diff
@@ -24,14 +25,23 @@ class NotFoundError < APIException
   belongs_to :review_assigned_from, class_name: 'Review', foreign_key: :review_id
   has_one :review_assigned_to, class_name: 'Review', foreign_key: :review_id
 
-  HISTORY_ELEMENTS_ASSIGNED_SUB_QUERY = <<-SQL
-    SELECT COUNT(history_elements.id) FROM history_elements
-    WHERE history_elements.op_object_id = reviews.id
-    AND history_elements.type = 'HistoryElement::ReviewAssigned'
-  SQL
+  scope :assigned, -> {
+    left_outer_joins(:history_elements_assigned).having('COUNT(history_elements.id) > 0').group('reviews.id')
```

Comment:
If you don't add the type comparison with 'HistoryElement::ReviewAssigned', any kind of history element will do that the review is "assigned"...

If you want to get rid of the plain SQL, maybe that way:
```ruby
scope :assigned, -> { joins(:history_elements).distinct.where(history_elements: { type: 'HistoryElement::ReviewAssigned' }) }
```
This generates:
```sql
SELECT DISTINCT `reviews`.* 
FROM `reviews` 
INNER JOIN `history_elements` 
ON `history_elements`.`op_object_id` = `reviews`.`id` AND 
`history_elements`.`type` IN ('HistoryElement::Review', 'HistoryElement::ReviewAccepted', 'HistoryElement::ReviewDeclined', 'HistoryElement::ReviewReopened', 'HistoryElement::ReviewObsoleted', 'HistoryElement::ReviewAssigned') 
WHERE `history_elements`.`type` = 'HistoryElement::ReviewAssigned'
```